mlflow/artifacts/__init__.py

```diff
@@ -0,0 +1,61 @@
+"""
+APIs for interacting with artifacts in MLflow
+"""
+import pathlib
+from typing import Optional
+
+from mlflow.exceptions import MlflowException
+from mlflow.protos.databricks_pb2 import INVALID_PARAMETER_VALUE
+from mlflow.tracking import _get_store
+from mlflow.tracking.artifact_utils import _download_artifact_from_uri, get_artifact_repository
+
+
+def download_artifacts(
+    artifact_uri: Optional[str] = None,
+    run_id: Optional[str] = None,
+    artifact_path: Optional[str] = None,
```

Comment:
While `run_id` and `artifact_path` are not strictly necessary, I've included them for parity with the existing `mlflow artifacts download` CLI.